src/tools/ScenarioMeasurement/Util/ProcessHelper.cs

```diff
@@ -135,7 +135,8 @@ public ProcessHelper(Logger logger)
                     if (!exited)
                     {
                         process.Kill();
-                        return (Result.CloseFailed, pid);
+                        Thread.Sleep(3000);
+                        return (Result.Success, pid);
```

Comment:
Why is this a success case now?

Reply:
This is dead code that I need to rip out. Good catch!